frontend/webpack.common.js

```
@@ -0,0 +1,80 @@
+var webpack = require('webpack');
```

ESLint

[error] 'webpack' is assigned a value but never used.

(no-unused-vars)